docs/user-guide/ze-use-cases.md

```diff
@@ -162,6 +162,77 @@ The following use cases demonstrate the various functionalities available in Zow
 
    ![Add Profile](../images/ze/ZE-mult-profiles.gif)
 
+### Filtering data set members
+
+Filter partitioned data set members in the **DATA SETS** tree view by **Date Modified** or **User ID**.
+
+#### Filtering all data set members under a specific profile
+
+1. In the **DATA SETS** tree, click on the **Filter** icon to the right of a profile, or right-click on a profile and select the **Filter PDS members…** option.
```

Comment:
Which option is the more common option? No need to describe both options to filter data sets